### ISSUE TITLE
Additional Prometheus Monitoring

### ISSUE THREAD

**Issue by stevenGravy** [gravitational/teleport#6238]
### What

Counter for failed tsh/ssh login attempts at the proxy level
Count of various failures for tsh/ssh attempt
  For example
     - Node not accessible/reachable
     - Cert mismatch
     - Cluster name not found
Time till  License expiry
Count of various rate limiters that were hit for
  - User sessions
  - Proxy sessions
  
Heartbeat connections received and missed by auth server
Count of user logins (similar to auth_generate_requests_total )

### How
Extend Prometheus monitoring

### Why
Unable to capture issues occuring in the environment easily without this. 

### Workaround
Extract some information from audit events logs.

**Comment by quinqu** [comment]
A few questions: 
- What scenario's regarding "cert mismatch" is this implying to keep track of? 
- Exactly which license should we be keeping track of until expiry? 
- Also, auth server doesn't know about heartbeat connections it missed because it misses them -- did you mean keep track of how many times the node failed to heartbeat?

**Comment by russjones** [comment]
@quinqu @fspmarshall Can we add per process TX/RX and also expose this via `tctl top`? This has come up with multiple customers and would be really useful in debugging.

**Comment by russjones** [comment]
@quinqu @fspmarshall I looked into this as well, might be hard for us to do right away, might be best to open another ticket for the TX/RX issue and do that later to unblock this work.

What do you two think?

**Comment by fspmarshall** [comment]
@russjones I'm good with that.

**Comment by russjones** [comment]
Fixed in https://github.com/gravitational/teleport/pull/6511.
